src/utils/dates.js

```diff
@@ -11,9 +11,26 @@ export function getFutureDate(offset) {
 	return new Date(Date.now() + offset * ONE_DAY_IN_MILLISECONDS);
 }
 
-export function getDaysBetweenDates(previousPurchase, currentDate) {
-	if (!currentDate) {
-		currentDate = Date.now();
+export function getDaysBetweenDates(previousPurchase, futurePurchase) {
+	const currentDate = new Date();
+	// console.log(previousPurchase, futurePurchase);
+	if (previousPurchase && futurePurchase) {
+		//console.log(`${previousPurchase && futurePurchase} previous and current`)
+		return Math.round(
+			Math.abs(
+				(previousPurchase.toDate() - futurePurchase.toDate()) /
+					ONE_DAY_IN_MILLISECONDS,
+			),
+		);
+	}
+	if (previousPurchase < currentDate) {
+		// console.log(previousPurchase < currentDate);
+		return Math.round(
+			(previousPurchase.toDate() - currentDate) / ONE_DAY_IN_MILLISECONDS,
+		);
+	} else {
+		return Math.round(
+			(currentDate - previousPurchase.toDate()) / ONE_DAY_IN_MILLISECONDS,
+		);
```

Comment:
Perhaps you would want to consider making this function cleaner?

This function takes two JavaScript Dates and returns the number of days that have passed between them
Input:  date1 - JS dates, date2 - JS dates
Output: daysBetweenDates - a positive int

So why not have something like:
```
function getDaysBetweenDates(date1, date2) {
	if no date1 or date2:
		set daysBetweenDates to 0;
	else 
		set daysBetweenDates to a rounded positive integer result of (date1 - date2) / ONE_DAY_IN_MILLISECONDS)
	return daysBetweenDates
}
```

Less room for error, and easier to reuse